python/Dockerfile

```
@@ -1,8 +1,17 @@
-FROM alpine
+FROM python:3.12-alpine
 
-RUN addgroup -S mercari && adduser -S trainee -G mercari
-# RUN chown -R trainee:mercari /path/to/db
+# setting working directory
+WORKDIR /app
 
-USER trainee
+# sqlite package
+RUN apk add --no-cache python3 py3-pip sqlite
 
-CMD ["python", "-V"]
+# install library
+RUN pip install --no-cache-dir fastapi uvicorn python-multipart
```

NIT: 今のところ大丈夫ですが、もしライブラリを変更しなきゃいけなくなったら、ここと『requirements.txt』を両方修正する必要があるので、ちょっと手間がかかると思います。

なので、

```suggestion
RUN pip install --no-cache-dir -r requirements.txt
```

という提案です。ただし、この方法を使う場合は、Dockerfileに COPY コマンドを追加する必要があります。

---

requirements.txtがあることに気付いていなかったです！ありがとうございます。訂正します。

---

訂正しました。確認よろしくお願いいたします。